src/components/global/Header/index.tsx

```diff
@@ -74,18 +89,46 @@ const Header = ({ currentPage = '' }) => {
                 Claim
               </HeaderNavItem>
             </NextLink>
-            <NextLink
-              href="https://snapshot.org/#/banklessvault.eth"
-              passHref={true}
-            >
+            <DropdownNavItem>
               <HeaderNavItem
                 tabIndex="0"
                 active={currentPage.includes('governance')}
-                onClick={() => setMobileNavIsActive(false)}
+                onClick={handleGovernanceDropdownClick}
               >
                 Governance
+                <DropdownCarot>
+                  <img
```

Comment:
Pull this out as a mini component so we can reuse it for both

Reply:
Good call!